src/Apps/Sell/Routes/PurchaseHistoryRoute.tsx

```diff
@@ -2,24 +2,49 @@ import * as React from "react"
 import * as Yup from "yup"
 import { DevDebug } from "Apps/Sell/Components/DevDebug"
 import { Formik } from "formik"
-import { Input, Text } from "@artsy/palette"
+import {
+  Column,
+  GridColumns,
+  Join,
+  Radio,
+  RadioGroup,
+  Select,
+  Spacer,
+  Text,
+} from "@artsy/palette"
 import { PurchaseHistoryRoute_submission$key } from "__generated__/PurchaseHistoryRoute_submission.graphql"
 import { SubmissionLayout } from "Apps/Sell/Components/SubmissionLayout"
 import { graphql, useFragment } from "react-relay"
 import { useSellFlowContext } from "Apps/Sell/SellFlowContext"
+import { __internal__useMatchMedia } from "Utils/Hooks/useMatchMedia"
 
 const FRAGMENT = graphql`
   fragment PurchaseHistoryRoute_submission on ConsignmentSubmission {
     provenance
+    signature
   }
 `
 
 const Schema = Yup.object().shape({
   provenance: Yup.string().trim(),
+  signature: Yup.boolean(),
 })
 
+export const PROVENANCE_LIST = [
+  "Purchased directly from gallery",
+  "Purchased directly from artist",
+  "Purchased at auction",
+  "Gift from the artist",
+  "Other",
+  "I don’t know",
+].map(provenance => ({
+  value: provenance,
+  text: provenance,
+}))
```

Comment:
For the provenance list here I used the same values as we use on the app. [In Figma for Web the texts are slightly different](https://www.figma.com/design/E6rVm1tiSI4PW3EzibKKZw/Submission-%26-Pre-Consignment-Management?node-id=2036-30651&m=dev). Should I use those from Figma or should we stay consistent and keep the same wording across platforms? cc @olerichter00 @MounirDhahri

Reply:
I think we should align the wording in the app and on the web. Maybe you can ask which version would be better and then update the values here or in the app to keep them consistent.